### PR TITLE
Add issue/PR templates

### PR DESCRIPTION
**Pull request by dracos**
[skip changelog]

### CODE REVIEW

**Comment by dracos** [comment]
@davea @struan @zarino @wrightmartin Anything obvious missing from PR checklist? Want anything extra/different in issue templates?

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Please check the following:
+
+- [ ] Whether this PR requires corresponding changes to documentation, or the FAQ;
+- [ ] All cobrand-specific commits start their commit message with the cobrand in square brackets;
+- [ ] Is new functionality tested? CodeCov will warn you about the diff coverage, but won’t complain about e.g. new files;
+- [ ] Have you updated the changelog? If this is not necessary, put square brackets around this: skip changelog
```
Comment (dracos) [inline .github/pull_request_template.md]: Accessibility

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Please check the contributing docs, and describe your pull request here.
+Please include any issues that are fixed, using "fixes" or "closes" so that they are auto-closed when the PR is merged.
+
```
Comment (struan) [inline .github/pull_request_template.md]: Screenshots so people know what it should look like?

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```
Comment (struan) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: Surely Mac OS here?

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```
Comment (struan) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: Worth noting that bugs for the mobile app should go in fixmystreet-mobile.

**Pull request merged**